buildscripts/resmokelib/testing/hooks/shard_merge.py

```diff
@@ -602,7 +605,8 @@ def _wait_for_migration_garbage_collection(self, migration_opts):  # noqa: D205,
                         donor_node_client = self._create_client(donor_node)
                         res = donor_node_client.config.command({
                             "count": "tenantMigrationDonors",
-                            "query": {"_id": bson.Binary(migration_opts.migration_id.bytes, 4)}
+                            "query": {"tenantIds": migration_opts.tenant_ids}
+                            # "query": {"_id": bson.Binary(migration_opts.migration_id.bytes, 4)}
```

Comment:
This was ported over from the shard split code, and I'm not sure if it's valid here.